### ISSUE TITLE
Test Case setting order is different than Robocop/style guide

### ISSUE THREAD

**Issue by emilezwiggelaar** [MarketSquare/robotframework-tidy#719]
I have a testcase in which I use [Template] and a [Setup], like this example:

Case One
    [Documentation]    datadriven testcase via test-template
    [Setup]    Log    Setup    console
    [Template]    A Template
    Just a keyword in testcase One
    [Teardown]    Log    Teardown    console

But when I run robotidy on my commandline it tells me that **[Template] should be before [Setup]**. (Well actually it 'fixes' the order automatically).

And then when I run robocop on my commandline I get this message: [W] 0927 'SETUP' is in wrong place of Test Case. Recommended order of elements in Test Cases: DOCUMENTATION, TAGS, TIMEOUT, **SETUP, TEMPLATE**, KEYWORD, TEARDOWN (test-case-section-out-of-order)
These styles seem to contradict each other.... in Slack I was told that Robocop uses the correct order so it would be nice to have robotidy use that same order...

(I know I can just configure tidy to use the same order as robocop but that is impractical)

**Comment by bhirsz** [comment]
I agree on it. I remember the discussion on it and I will adjust so both Robocop and Robotidy uses the same order. 

Ideally though it should be just robocop rule with option to 'fix' the issue. I will look into it in the future Robocop first ('Robocop 2' or merged both tools). If not, I will release it in robotidy

**Comment by bhirsz** [comment]
Adjusted formatter order to match linter & style guide recommendation - done in the linked PR. It will be releases as part of Robocop 6.0.
